code/modules/cm_marines/equipment/mortar/mortar_shells.dm

```diff
@@ -59,6 +59,33 @@
 	playsound(T, 'sound/weapons/gun_flare.ogg', 50, 1, 4)
 	deploy_camera(T)
 
+/obj/item/mortar_shell/cluster
+	name = "\improper 80mm cluster mortar shell"
+	desc = "An 80mm mortar shell, loaded with cluster munitions."
+	icon_state = "mortar_ammo_cluster"
+
+	var/total_amount = 8
+	var/instant_amount = 3
+
+/obj/item/mortar_shell/cluster/detonate(var/turf/T)
+	start_cluster(T)
+
+/obj/item/mortar_shell/cluster/proc/start_cluster(turf/target)
+	set waitfor = 0
+
+	var/range_num = 7
```

Comment:
might want to consider reducing range ig

Reply:
i think making it concentrated on a small area would make it too much of a choke-buster, the bigger range makes it more balanced in that way. maybe we should TM and see?